pepper-apis/dsm-core/src/main/java/org/broadinstitute/dsm/util/DSMConfig.java

```diff
@@ -3,35 +3,47 @@
 import com.typesafe.config.Config;
 import lombok.NonNull;
 
+/**
+ * Wrapper around the config used for DSM
+ */
 public class DSMConfig {
 
     private static Config config;
 
-    public DSMConfig(Config config) {
-        this.config = config;
+    public static void setConfig(Config cfg) {
+        config = cfg;
     }
 
     public static String getSqlFromConfig(@NonNull String queryName) {
+        checkNullConfig();
+        if (!config.hasPath(queryName)) {
+            throw new RuntimeException("Conf is missing query named " + queryName);
+        }
+        return config.getString(queryName);
+    }
+
+    private static void checkNullConfig() {
         if (config == null) {
             throw new RuntimeException("Config is null ");
         }
+    }
 
-        if (!config.hasPath(queryName)) {
-            throw new RuntimeException("Conf is missing query named " + queryName);
-        }
+    public static int getIntFromConfig(String path) {
+        checkNullConfig();
```

Comment:
Small refactor to write a checkNullConfig method instead of copy pasting the check.